annotations/spar.md

```diff
@@ -0,0 +1,24 @@
+**_spar_** (English); _rundholt_ (Danish); _Rundholz_ (German)
+**_spar_** (English); _spir_ (Danish); _Spiere_ (German)
+
+_**áss** m., pl. ásar_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o4934https://onp.ku.dk/onp/onp.php?o4934)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=5136)]  
+
+ A long wooden pole connected to the sail (_segl_), used to angle the sail into the wind creating a technique called tacking (_hals_).    
+
+<div align="center">
+  
+<img width="470" alt="Screenshot 2024-12-04 at 8 53 06 pm" src="https://github.com/user-attachments/assets/949c9959-9fc1-4166-88b5-47775688de48">
+  
+  Gotland Picture Stone (The Picture Stone from Hunninge at Klinte, Viking Archaeology)
+
+</div>
+
+  Use of the spar reduces strain on the side of a ship, by spreading the pressure from the sail over to the opposite side, where the forces can be spread over several frames (Bishcoff, Vibeke). 2). The use of beitiass is also known from iconographie sources as on the Gotland picture stone from Klinte Hunninge and from ethnographic boats from northwestern Iceland (Imer 2004, 58; Andersen and Andersen
```

Comment:
This is well written, but the Imer 2004 and Andersen and Andersen references are not included below
```suggestion
  Use of the spar reduces strain on the side of a ship, by spreading the pressure from the sail over to the opposite side, where the forces can be spread over several frames (Bishcoff 2017, 2). The use of the beitiáss is also known from iconographic sources as on the Gotland picture stone from Klinte Hunninge and from ethnographic boats from northwestern Iceland (Imer 2004, 58; Andersen and Andersen
```